LOAN/LoanTypes/LoanProducts.rdf

```diff
@@ -52,35 +52,66 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
+				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;specifies"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-col;LoanCollateral"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;AutoLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">auto loan purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;BridgingLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">bridging loan purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;ConstructionLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">construction loan purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;CreditCardReceivablePurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">credit card receivable purpose</rdfs:label>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;HomeEquityLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-prod;LoanPurpose"/>
+		<rdfs:label xml:lang="en">home equity loan purpose</rdfs:label>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProductDisclosureRight">
 		<rdfs:subClassOf rdf:resource="&fibo-loan-loan-reg;ProductDisclosureRight"/>
 		<rdfs:label xml:lang="en">loan product disclosure right</rdfs:label>
 		<skos:definition xml:lang="en">Right to fair representation of the facts about Loan products before entrering into agreements for same.</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Objective"/>
+		<rdfs:label xml:lang="en">loan purpose</rdfs:label>
+		<skos:definition xml:lang="en">A selection of different types of loan purpose, being the purpose for which and manner in which loan (credit) draw-down amounts are to be used. This shows the purpose for which credit is to be used, and implies certain kinds of fact that relate to that specific type of loan e.g. mortgages. These are also identified as tranche types in tranches of a credit facility.</skos:definition>
```

Comment:
this definition is pretty baffling (why does the loan purpose require "draw down amounts"?)  and not ISO 704 compliant.

Reply:
didn't work on that, just moved this from an ontology I deleted and there were references ... stay tuned for another pass.